{"uuid":"1f235198-9442-41e2-8872-016971b3cb13","historyId":null,"status":"broken","statusDetails":{"message":"Oops...we found an error preparing this test file:\n\n  > cypress\\e2e\\TestSuite\\Opportunity\\createOpportunity.cy.js\n\n","trace":"\n\nError: Webpack Compilation Error\nModule not found: Error: Can't resolve '../../../pages/Loginpagedev' in 'D:\\cypress\\UserJourney_Automation\\cypress\\e2e\\TestSuite\\Opportunity'\n\nModule not found: Error: Can't resolve '../../../pages/CreateOpportunity' in 'D:\\cypress\\UserJourney_Automation\\cypress\\e2e\\TestSuite\\Opportunity'\n\nModule not found: Error: Can't resolve './BasePage' in 'D:\\cypress\\UserJourney_Automation\\cypress\\pages'\n    at handle (C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\@cypress\\webpack-preprocessor\\dist\\index.js:212:23)\n    at finalCallback (C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:441:32)\n    at C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:505:17\n    at C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\HookWebpackError.js:68:3\n    at Hook.eval [as callAsync] (eval at create (C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:18:14)\n    at Cache.storeBuildDependencies (C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Cache.js:122:37)\n    at C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:501:19\n    at Hook.eval [as callAsync] (eval at create (C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:18:14)\n    at C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:498:23\n    at Compiler.emitRecords (C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:919:5)\n    at C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:490:11\n    at C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:885:14\n    at Hook.eval [as callAsync] (eval at create (C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:18:14)\n    at C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:882:27\n    at C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\neo-async\\async.js:2818:7\n    at done (C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\neo-async\\async.js:3522:9)\n    at Hook.eval [as callAsync] (eval at create (C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:18:14)\n    at C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:736:33\n    at C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\graceful-fs\\graceful-fs.js:143:16\n    at C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\graceful-fs\\graceful-fs.js:143:16\n    at C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\graceful-fs\\graceful-fs.js:143:16\n    at C:\\Users\\MonishKumar.N\\AppData\\Local\\Cypress\\Cache\\14.0.2\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\graceful-fs\\graceful-fs.js:61:14\n    at FSReqCallback.oncomplete (node:fs:187:23)\n\nThis occurred while Cypress was compiling and bundling your test code. This is usually caused by:\n\n- A missing file or dependency\n- A syntax error in the file or one of its dependencies\n\nFix the error in your code and re-run your tests."},"stage":"finished","attachments":[],"parameters":[],"labels":[{"name":"suite","value":"createOpportunity.cy.js"}],"links":[],"start":null,"name":"Oops...we found an error preparing this test file: createOpportunity.cy.js","fullName":"Oops...we found an error preparing this test file: createOpportunity.cy.js","stop":null}